curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md

```diff
@@ -80,9 +80,17 @@ assert.deepEqual(tokenize(testStr2, '&', '@'), res2);
 const testStr1 = 'one^|uno||three^^^^|four^^^|^cuatro|';
 const res1 = ['one|uno', '', 'three^^', 'four^|cuatro', ''];
 
-// TODO add more tests
 const testStr2 = 'a@&bcd&ef&&@@hi';
 const res2 = ['a&bcd', 'ef', '', '@hi'];
+
+const testStr3 = 'ab^cd^ef^^gh';
+const res3 = ['ab^cd^ef^gh'];
+
```

Comment:
you only added some strints and arrays, doing nothing with them, as such, no new test has been added.